public/resources/remix/style.less

```diff
@@ -272,3 +272,22 @@
     }
   }
 }
+/* move the remix button to the left side for AFrame projects only*/
+.a-body .details-bar.touch-mode {
+  right: auto !important;
+  left: 6px !important;
+    
+  .thimble-button {
+    left: 0px !important;
```

Comment:
@ZenanZha for next time, a tip: when doing `0` in CSS don't put any units, that is: `0` (correct) vs. `0px` (incorrect).  It's not a huge deal, and not worth blocking this any further, but fyi.